### PR TITLE
Changes to transform for the first phase transformations

### PR DESCRIPTION
**Pull request by silveroliver**

### CODE REVIEW

File: ontologies/WIP/sierratransformable [hunk 1]
```diff
@@ -100,84 +100,136 @@
...
+   *           "value": "2005016805",
+   *           "type": "Identifier",
+   *           "identifierScheme": "lc-names"
+   *        }
+   *        ]
+   *     } 
+   *     ]
+   *
+   
+   /* Populate wwork:contribution. Rules:
```
Comment (alexwlchan) [inline ontologies/WIP/sierratransformable]: Is it contribution or contributors? Example below users "contributors".

File: ontologies/WIP/sierratransformable [hunk 2]
```diff
@@ -100,84 +100,136 @@
...
-   * 1. For all bibliographic records use "marcTag": "655"
-   * 2. If more than one subfield is populated then populate platform "label" with concatenated subfields in citation/list order and type as "QualifiedConcept".
-   * 3. If only "a" subfield is populated type as "Concept".   
-   * 4. If 655 is populated with a indentified concept from an authority (as indicated by use of second indicator) then populate id with Wellcome id for concept
-   * 5. populate platform "concept" with subfield "a".
-   * 6. When this concept has associated identifiers populate "identifiers" with value (subfield 0) and identity scheme (second indicator or if indicator 7 look to subfield 2) Secound indicator 0=lcsh 2=mesh
-   * 8. For each subfield, for subfields a-z ingnore 0-7, populate "qualifiers" with a "label", type "Concept" (subfield "y" type as "Period" and type z as "Place") and populate "qualiferType" with the subfield name kebab-case.
+   * 
+   * 1. For all bibliographic records use "marcTag": "655".
+   * 2. If one of subfield v,x,y,z is populated. Populate platform "label" with concatenated subfields v,x,y,z in order and type as "QualifiedConcept".
```
Comment (alexwlchan) [inline ontologies/WIP/sierratransformable]: Just joined together, no extra punctuation?

File: ontologies/WIP/sierratransformable (hunk 1 above)
Comment (silveroliver) [reply ontologies/WIP/sierratransformable]: Changing top to accurately state contributors

File: ontologies/WIP/sierratransformable (hunk 2 above)
Comment (silveroliver) [reply ontologies/WIP/sierratransformable]: Originally we had hyphens which I removed because unconfirmed. Will work out what we will use and add today.

**Pull request merged**